app/Models/Post.php

```diff
@@ -53,7 +53,7 @@ class Post extends Model
      *
      * @var array
      */
-    public static $indexes = ['id', 'signup_id', 'campaign_id', 'type', 'action', 'northstar_id', 'status', 'created_at'];
+    public static $indexes = ['id', 'signup_id', 'campaign_id', 'type', 'action', 'northstar_id', 'status', 'created_at', 'source'];
```

Comment:
Did we already add a corresponding database index on this field?

Reply:
@DFurnes yeah - looks like this was already an index in the `posts` and `signups` tables!

Reply:
Hmm, I don't see that index when looking at those tables on QA & am having a hard time finding where we set that in a migration. Can you point me in the right direction?

Reply:
oh you are totally right - sorry @DFurnes ! It was already set on my local weirdly so that is why I thought it was an index already... I'll add this migration in now!

Reply:
Spooky! I'd recommend running `php artisan migrate:fresh --seed` if you haven't lately.

Reply:
I'm having trouble getting a `filter[type]` query parameter to work on a `GET /v3/posts` request... and can't tell if it's an error in on my end or a bug? I'm using the `gambit` OAuth client.

Reply:
ah interesting, what type are you trying to filter by @aaronschachter ?

Reply:
🤦‍♂️ This is working for me, sorry for a false flag! I sometimes forget to use the `filter[fieldName]` syntax 🥇

Reply:
it is a confusing one! Glad it is working ! :)